docker-compose.yml

```diff
@@ -297,6 +297,23 @@ services:
     volumes:
       - pecan:/data
 
+# ----------------------------------------------------------------------
+# Shiny Apps
+# ----------------------------------------------------------------------
+  # PEcAn DB Sync visualization
+  maespa:
```

Comment:
this naming seems odd — why maespa here?

Reply:
bad copy/paste